.github/dependabot.yml

```diff
@@ -1,69 +1,20 @@
 version: 2
 updates:
 - package-ecosystem: npm
-  directory: "/"
+  directory: "/addon"
   schedule:
     interval: daily
     time: "02:00"
-  open-pull-requests-limit: 10
-  ignore:
-  - dependency-name: ember-cli
-    versions:
-    - 3.24.0
-    - 3.25.0
-    - 3.25.1
-    - 3.25.2
-    - 3.25.3
-    - 3.26.0
-  - dependency-name: eslint
-    versions:
-    - 7.20.0
-    - 7.21.0
-    - 7.22.0
-    - 7.23.0
-    - 7.24.0
-  - dependency-name: release-it
-    versions:
-    - 14.3.0
-    - 14.4.0
-    - 14.4.1
-    - 14.5.0
-    - 14.5.1
-  - dependency-name: y18n
-    versions:
-    - 4.0.1
-  - dependency-name: ember-template-lint
-    versions:
-    - 2.18.1
-    - 2.19.0
-    - 2.20.0
-    - 2.21.0
-    - 3.0.0
-    - 3.0.1
-    - 3.1.1
-    - 3.2.0
-  - dependency-name: ember-source
-    versions:
-    - 3.24.1
-    - 3.25.0
-    - 3.25.1
-    - 3.25.2
-    - 3.25.3
-    - 3.26.0
-  - dependency-name: eslint-plugin-ember
-    versions:
-    - 10.1.2
-    - 10.2.0
-    - 10.3.0
-  - dependency-name: ember-cli-htmlbars
-    versions:
-    - 5.3.2
-    - 5.4.0
-    - 5.6.2
-    - 5.6.3
-    - 5.6.4
-    - 5.6.5
-    - 5.7.0
-  - dependency-name: marked
-    versions:
-    - 1.2.9
+  open-pull-requests-limit: 4
+- package-ecosystem: npm
+  directory: "/test-app"
+  schedule:
+    interval: daily
+    time: "02:00"
+  open-pull-requests-limit: 3
+- package-ecosystem: npm
+  directory: "/docs"
+  schedule:
+    interval: daily
+    time: "02:00"
+  open-pull-requests-limit: 3
```

Comment:
Delete docs portion?

Reply:
@lukemelia deleted. This will be needed for part 2 when docs app will be extracted into separate app on `/docs` folder